scannermsbuild.properties

```
@@ -16,6 +16,15 @@ flavors.core20.label=.NET Core 2.1
 flavors.core31.label=.NET Core 3.1
 flavors.framework46.label=.NET Framework 4.6+
 
+5.9.0.description=.NET 7 Bug Fixes and preperation for Fast PR Analysis 
```

Small typo in "preperation"
```suggestion
5.9.0.description=.NET 7 bug fixes and preparation for fast PR analysis 
```